api/internal/broker/channeled.go

```diff
@@ -55,9 +56,22 @@ func (broker *ChannelBroker) Listen() {
 		case s := <-broker.NewClients:
 			broker.Clients[s] = true
 			logrus.Infof("🟢 Client added. %d registered clients", len(broker.Clients))
+
+			event := sse.Event{
+				EventType: sse.CLIENT_CONNECT_CHANGE,
+				Payload:   fmt.Sprint(len(broker.Clients)),
```

Comment:
this does not work , because i user can connect with multiple tabs, every tab counts as separate client on this level, i also did a pull request concerning that feature xD

Reply:
You are absolutely right :) did not think we'll enough about this

Reply:
i approved yours and reverted my changes in the backend